service/storage_service.cc

```diff
@@ -2526,14 +2540,7 @@ future<> storage_service::removenode(locator::host_id host_id, std::list<locator
                     }).get();
                     on_streaming_finished();
 
-
-                    // Step 6: Announce the node has left
-                    ss._gossiper.advertise_token_removed(endpoint, host_id).get();
-                    std::unordered_set<token> tmp(tokens.begin(), tokens.end());
-                    ss.excise(std::move(tmp), endpoint).get();
-                    removed_from_token_ring = true;
-
-                    // Step 7: Finish token movement
+                    // Step 6: Finish token movement
```

Comment:
This comment should be updated, after this patch we're not finishing token movement at this step.

Reply:
Don't know tbh. It's not like this comment was of much use anyway. Let's leave it as it is then.